### ISSUE TITLE
Editing LXQt menu

### ISSUE THREAD

**Issue by ghost** [lxqt/lxqt#255]
I'd like to edit the LXQt menu. In particular I'd like to remove/hide applications that I never use. I also don't really like applications being divided into categories. What do you think?

**Comment by Hacktivista** [comment]
Is there any advance with this? May we fork another menu-editor at least?

**Comment by Vladimir-csp** [comment]
https://github.com/lxde/lxqt/issues/820#issuecomment-139810933

**Comment by grepwood** [comment]
1 year since high prio. Cmon, I want to make a bunch of convenient shortcuts without installing MATE and alacarte ;)

**Comment by jfk88** [comment]
> Categories are here to stay but editing the menu is a planned feature of the advanced menu.
Why categories are here to stay? I agree with fturco this menu app divided into categories is not really straightforward. For instance, I have Featherpad in "Utility" whereas it's more intuitive to me to be with LibreOffice. More broadly, as we can't determine where an app goes, the role of the classification might play against itself as we have to guess where it's been located. Wouldn't it be cleaner and easier to have a windows-like option?

**Comment by wxl** [comment]
Categories are an inherent part of the Freedesktop (XDG for old fogies) spec. They're there to stay unless you can convince those folks to get rid of them (won't happen).

**Comment by tsujan** [comment]
There may be a misunderstanding here.

Everything in the main menu can be changed by the user. This report is about making an LXQt/Qt app that does so; otherwise, those who are familiar with desktop entries can already do it easily, without needing an extra app.

**Comment by jfk88** [comment]
> Categories are an inherent part of the Freedesktop (XDG for old fogies) spec. They're there to stay unless you can convince those folks to get rid of them (won't happen).

OK I see. Thank you!

**Comment by jfk88** [comment]
> There may be a misunderstanding here.
> 
> Everything in the main menu can be changed by the user. This report is about making an LXQt/Qt app that does so; otherwise, those who are familiar with desktop entries can already do it easily, without needing an extra app.

Yes but the original discussion was also asking for the removal of those categories.
